internal/view/yaml.go

```diff
@@ -25,7 +24,7 @@ const (
 )
 
 func colorizeYAML(style config.Yaml, raw string) string {
-	lines := strings.Split(tview.Escape(raw), "\n")
+	lines := strings.Split(raw, "\n")
```

Comment:
The escape here is important as it will trip the view if their are "[]" in the yaml and will cause the display to bomb.

Reply:
@derailed thanks! Let me try it again...